.github/workflows/update-cni-plugins-version.yml

```diff
@@ -0,0 +1,70 @@
+name: "update-cni-plugins-version"
```

Comment:
It will only be ISO temporarily, once https://github.com/kubernetes/minikube/pull/16069 is merged we'll also need to update the kicbase as well. I can update it until then if you would like though